ui/src/main/java/com/midtrans/sdk/ui/views/transaction/TransactionActivity.java

```diff
@@ -235,11 +238,41 @@ private void showPaymentActivity(PaymentMethodModel paymentMethod) {
             case PaymentType.GCI:
                 startGiftCardPaymentFlow();
                 break;
+            case PaymentType.TELKOMSEL_CASH:
+                startTelkomselCashPaymentFlow();
+                break;
+            case PaymentType.XL_TUNAI:
+                startXlTunaiPaymentFlow();
+                break;
+            case PaymentType.INDOSAT_DOMPETKU:
+                startIndosatDompetkuPaymentFlow();
+                break;
             default:
                 break;
         }
     }
 
+    private void startIndosatDompetkuPaymentFlow() {
+        Intent intent = new Intent(this, IndosatDompetkuActivity.class);
+        startActivityForResult(intent, Constants.INTENT_CODE_PAYMENT);
+    }
+
+    private void startXlTunaiPaymentFlow() {
+        Intent intent = new Intent(this, XlTunaiPaymentActivity.class);
+        startActivityForResult(intent, Constants.INTENT_CODE_PAYMENT);
+    }
+
+    private void startBankTransferPaymentFlow() {
+        Intent bankTransferIntent = new Intent(this, BankTransferListActivity.class);
+        bankTransferIntent.putStringArrayListExtra(BankTransferListActivity.ARGS_BANK_LIST, new ArrayList<>(presenter.getBankList()));
+        startActivityForResult(bankTransferIntent, Constants.INTENT_CODE_PAYMENT);
+    }
+
+    private void startTelkomselCashPaymentFlow() {
```

Comment:
Please put this after `startGiftCardPaymentFlow()` method declaration so it has same order as the order its called